assets/js/modules/analytics-4/components/dashboard/DashboardAllTrafficWidgetGA4/UserDimensionsPieChart.js

```diff
@@ -171,15 +171,16 @@ export default function UserDimensionsPieChart( props ) {
 		maxSlices: 5,
 		withOthers: true,
 		tooltipCallback: ( row, previousDateRangeRow, rowData ) => {
+			const rowValue = row?.metricValues?.[ 0 ]?.value || 0;
 			const difference =
-				previousDateRangeRow.metricValues?.[ 0 ].value > 0
-					? ( row.metricValues?.[ 0 ].value * 100 ) /
-							previousDateRangeRow.metricValues?.[ 0 ].value -
+				previousDateRangeRow?.metricValues?.[ 0 ]?.value > 0
+					? ( rowValue * 100 ) /
+							previousDateRangeRow?.metricValues?.[ 0 ]?.value -
```

Comment:
This one should also be casted to `0` if its value is `undefined` because this division will return `NaN` otherwise.

```js
100 / undefined -> NaN
```

Reply:
@eugene-manuilov this is inside of a check already: 

```
previousDateRangeRow?.metricValues?.[ 0 ]?.value > 0
```

So it should never be undefined?